libs/sdk-ui-filters/src/AttributeFilter/AttributeDropdown/AttributeDropdownBody.tsx

```diff
@@ -6,8 +6,30 @@ import { AttributeDropdownList } from "./AttributeDropdownList";
 import { AttributeDropdownButtons } from "./AttributeDropdownButtons";
 import { AttributeListItem } from "./types";
 import { AttributeDropdownItemsFilteredBody } from "./AttributeDropdownItemsFilteredBody";
+import { ILegacyMultiSelectListItemProps } from "@gooddata/sdk-ui-kit";
+import { ObjRef } from "@gooddata/sdk-model";
 
-interface IAttributeDropdownBodyProps {
+/**
+ * @public
+ */
+export interface IAttributeDropdownBodyExtendedProps extends IAttributeDropdownBodyProps {
```

Comment:
I believe this type should "live" in AttributeFilterButton and be called something like `IRenderAttributeDropdownProps`. Because this type is not used in this file anywhere.